python/cuml/test/test_random_forest.py

```diff
@@ -210,6 +213,11 @@ def test_rf_regression_default(datatype, column_info, nrows):
     cu_r2 = r2_score(y_test, cu_preds, convert_dtype=datatype)
     fil_r2 = r2_score(y_test, fil_preds, convert_dtype=datatype)
 
+    # score function should be equivalent
+    score_mse = cuml_model.score(X_test, y_test)
+    manual_mse = ((fil_preds - y_test)**2).mean()
+    assert manual_mse == pytest.approx(score_mse)
```

Comment:
Same question as above, can't remember why this checks are new in this PR

Reply:
As above.